### PR TITLE
Add list of values and defaults for wootricSettings

### PR DESCRIPTION
**Pull request by marinhero**
Add a description of the wootricSettings object and it's officially supported values.

Trello: https://trello.com/c/6ZVI6PMg/3874-update-docs-for-wootricsettings-object-as-requested-by-dialpad

### CODE REVIEW

File: source/includes/_install_js_intro.md [hunk 1]
```diff
@@ -53,6 +53,41 @@ data.
...
+aria | null |Enable extended aria support for our survey modal
+created_at | null | The created_at date in your app for the end_user you are attempting to survey
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page
+language | null | The language code of the survey
+product_name | depends on the language | Your product name. 
+properties  | null | The custom properties of the end_user you are attempting to survey
+twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment
+wootric_close_position | left | The position of the close button of your survey
+wootric_recommend_target | depens of the language| Your audience
```
Comment (prabhatjha) [inline source/includes/_install_js_intro.md]:
```suggestion
wootric_recommend_target | depends on the language| Your audience
```

File: source/includes/_install_js_intro.md [hunk 2]
```diff
@@ -53,6 +53,41 @@ data.
...
+
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Mandatory
+aria | null |Enable extended aria support for our survey modal
+created_at | null | The created_at date in your app for the end_user you are attempting to survey
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page
+language | null | The language code of the survey
+product_name | depends on the language | Your product name.
```
Comment (prabhatjha) [inline source/includes/_install_js_intro.md:85]: Since we are making an effort, it's better to show what it affects the survey with an example IMO.

Comment (marinhero) [reply source/includes/_install_js_intro.md:85]: Great idea, I'll make sure I add an image of what this generates

Comment (marinhero) [reply source/includes/_install_js_intro.md:85]: example-one
example-two

**Comment by marinhero** [comment]
Example images have been added!

File: source/includes/_install_js_intro.md [hunk 3]
```diff
@@ -53,6 +53,48 @@ data.
...
+};
+```
+
+This is a list of the values you can set into your wootricSettings object and it's defaults.
+Keep in mind that your code settings will always override whatever you set in your Wootric Account Settings page.
+
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Mandatory
+aria | null |Enable extended accessibility features (high contrast and extended aria support)
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
aria | null | Enable extended accessibility features (high contrast and extended aria support)
```

File: source/includes/_install_js_intro.md [hunk 4]
```diff
@@ -53,6 +53,48 @@ data.
...
+aria | null | Enable extended accessibility features (high contrast and extended aria support)
+created_at | null | The created_at date in your app for the end_user you are attempting to survey
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page
+language | null | The language code of the survey
+product_name | depends on the language | Your product name. 
+properties  | null | The custom properties of the end_user you are attempting to survey
+twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment
+wootric_close_position | left | The position of the close button of your survey
+wootric_recommend_target | depens of the language| Your audience
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
wootric_recommend_target | depends of the language | Your audience
```

File: source/includes/_install_js_intro.md [hunk 5]
```diff
@@ -53,6 +53,48 @@ data.
...
+  language: 'EN',
+  product_name: 'your product name',
+  properties : { country: 'MX' },
+  twitter_account: 'test',
+  wootric_close_position: 'left',
+  wootric_recommend_target: 'your audience'
+};
+```
+
+This is a list of the values you can set into your wootricSettings object and it's defaults.
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
This is a list of the values you can set into your wootricSettings object and its defaults.
```

File: source/includes/_install_js_intro.md [hunk 6]
```diff
@@ -53,6 +53,48 @@ data.
...
+Keep in mind that your code settings will always override whatever you set in your Wootric Account Settings page.
+
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Mandatory
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
account_token | null | Your account token. Required.
```

File: source/includes/_install_js_intro.md [hunk 7]
```diff
@@ -53,6 +53,48 @@ data.
...
+aria | null | Enable extended accessibility features (high contrast and extended aria support)
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
aria | null | Enable extended accessibility features such as high contrast and extended aria support.
```

File: source/includes/_install_js_intro.md [hunk 8]
```diff
@@ -53,6 +53,48 @@ data.
...
+created_at | null | The created_at date in your app for the end_user you are attempting to survey
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page
+language | null | The language code of the survey
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
language | null | The language code of the survey.
```

File: source/includes/_install_js_intro.md [hunk 9]
```diff
@@ -53,6 +53,48 @@ data.
...
+This is a list of the values you can set into your wootricSettings object and it's defaults.
+Keep in mind that your code settings will always override whatever you set in your Wootric Account Settings page.
+
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Mandatory
+aria | null | Enable extended accessibility features (high contrast and extended aria support)
+created_at | null | The created_at date in your app for the end_user you are attempting to survey
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page.
```

File: source/includes/_install_js_intro.md [hunk 10]
```diff
@@ -53,6 +53,48 @@ data.
...
+```
+
+This is a list of the values you can set into your wootricSettings object and it's defaults.
+Keep in mind that your code settings will always override whatever you set in your Wootric Account Settings page.
+
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Mandatory
+aria | null | Enable extended accessibility features (high contrast and extended aria support)
+created_at | null | The created_at date in your app for the end_user you are attempting to survey
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
created_at | null | The created_at date in your app for the end_user you are attempting to survey.
```

File: source/includes/_install_js_intro.md [hunk 11]
```diff
@@ -53,6 +53,48 @@ data.
...
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Required.
+aria | null | Enable extended accessibility features such as high contrast and extended aria support.
+created_at | null | The created_at date in your app for the end_user you are attempting to survey.
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page.
+language | null | The language code of the survey.
+product_name | depends on the language | Your product name. 
+properties  | null | The custom properties of the end_user you are attempting to survey
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
properties  | null | The custom properties of the end_user you are attempting to survey.
```

File: source/includes/_install_js_intro.md [hunk 12]
```diff
@@ -53,6 +53,48 @@ data.
...
+twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment.
```

File: source/includes/_install_js_intro.md [hunk 13]
```diff
@@ -53,6 +53,48 @@ data.
...
+wootric_close_position | left | The position of the close button of your survey
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
wootric_close_position | left | The position of the close button of your survey.
```

File: source/includes/_install_js_intro.md [hunk 14]
```diff
@@ -53,6 +53,48 @@ data.
...
+wootric_recommend_target | depends of the language | Your audience
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
wootric_recommend_target | depends of the language | Your audience.
```

Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
wootric_recommend_target | depends on the language | Your audience
```

File: source/includes/_install_js_intro.md [hunk 15]
```diff
@@ -53,6 +53,48 @@ data.
...
+Variable | Default | Description
+---- | ---- | -------
+account_token | null | Your account token. Required.
+aria | null | Enable extended accessibility features such as high contrast and extended aria support.
+created_at | null | The created_at date in your app for the end_user you are attempting to survey.
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page.
+language | null | The language code of the survey.
+product_name | depends on the language | Your product name. 
+properties  | null | The custom properties of the end_user you are attempting to survey.
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
properties | null | The custom properties of the end_user you are attempting to survey.
```

File: source/includes/_install_js_intro.md [hunk 16]
```diff
@@ -53,6 +53,48 @@ data.
...
+aria | null | Enable extended accessibility features such as high contrast and extended aria support.
+created_at | null | The created_at date in your app for the end_user you are attempting to survey.
+email | null | The email of the end_user you are attempting to survey
+facebook_page | null | A Facebook link you can use to refer promoters to your Facebook page.
+language | null | The language code of the survey.
+product_name | depends on the language | Your product name. 
+properties | null | The custom properties of the end_user you are attempting to survey.
+twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment.
+wootric_close_position | left | The position of the close button of your survey.
+wootric_recommend_target | depends of the language | Your audience.
```
Comment (hachibu) [inline source/includes/_install_js_intro.md]:
```suggestion
wootric_recommend_target | depends on the language | Your audience.
```

**Comment by hachibu** [comment]
English looks good to me. I'll let @prabhatjha give you the final 👍

**Comment by marinhero** [comment]
Thanks Ray-o! I'll wait for bossman to reply :)

**Review by hachibu — APPROVED** [review]
Booted this locally and I see the images.

**Comment by hachibu** [comment]
Don't forget to squash.

**Pull request merged**